### PR TITLE
Docker Support

### PR DESCRIPTION
**Pull request by hggggggggggggggg**
Hello, sirs. I have added docker support to kregfile. Kindly pull the commit to master.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version: '3.3'
+
+services:
+  kregfile:
+    image: local/kregfile
+    build: .
+    container_name: kregfile
+    volumes:
+      - type: bind
+        source: ./.config.json
```
Comment (RealDolos) [inline docker-compose.yml:12]: >file not found

**Review by RealDolos — CHANGES_REQUESTED** [review]
Sirs, you have many mistakes, sirs

**Comment by hggggggggggggggg** [comment]
Hello, sirs. I do not know what you mean. I have reopened the pull request.

**Comment by hggggggggggggggg** [comment]
I made a change to the Dockerfile to simplify the redis docker-compose. I also corrected the comments in the docker-compose.

**Pull request closed without merging**